tests/test_single_topology.py

```diff
@@ -95,17 +94,16 @@ def test_find_dummy_groups_and_anchors():
 
     core_pairs = np.array([[1, 2], [2, 1], [3, 0]])
 
-    dgs, jks = single_topology.find_dummy_groups_and_anchors(mol_a, mol_b, core_pairs[:, 0], core_pairs[:, 1])
+    dgs, jks = single_topology.find_dummy_groups_and_anchors(mol_b, core_pairs[:, 1])
     assert dgs == [{3}]
     assert jks == [(2, 1)]
 
     # angle should swap
     core_pairs = np.array([[1, 2], [2, 0], [3, 1]])
 
-    with pytest.warns(CoreBondChangeWarning):
-        dgs, jks = single_topology.find_dummy_groups_and_anchors(mol_a, mol_b, core_pairs[:, 0], core_pairs[:, 1])
-        assert dgs == [{3}]
-        assert jks == [(2, None)]
```

Comment:
This is testing the following case:
![test-atom-map](https://user-images.githubusercontent.com/319411/235203705-9025177d-d32e-475a-9582-10801874b7e4.png)

Reply:
Currently, the angle 3-2-1 is disallowed because the blue-green bond is not present in mol_a. With these changes, it is allowed.

Reply:
Hm after staring at this a bit more - we will leave this angle term on at all values of lambda - wouldn't it flop around considerably? (The dummy anchor terms are not alchemically turned off).